include/onnxruntime/core/common/logging/severity.h

```diff
@@ -13,10 +13,11 @@ enum class Severity {
   kINFO = 1,
   kWARNING = 2,
   kERROR = 3,
-  kFATAL = 4
+  kFATAL = 4,
+  kTELEM = 5,
```

Comment:
nit: Telemetry seems like a usage of the log level, but not necessarily the only one. Would kALWAYS be more generic? 

If this is short term and we're going to remove it and prefer a telemetry provider based system in the future it doesn't really matter.